README.md

```diff
@@ -4,56 +4,24 @@ Dart Code Lab Samples
 
 These are small Dart samples used by the following codelabs:
 
-* [Avast Ye Pirates][ng2-codelab] where you learn to build an Angular 2 web app.
-* [Original pirate codelab written using dart:html][client-codelab], where you learn to build a web app. This lab should take about an hour to complete.
-* [Beware the Nest o' Pirates][server-codelab] which shows you how to write a RESTful Dart server.
+* [AngularDart Components Lottery Simulator][] Beautify a web app by converting its vanilla HTML elements to Material
+  Design components that are widely used in Google’s Dart apps.
+* (**Deprecated**) [Avast Ye Pirates][ng2-codelab] where you learn to build an Angular 2 web app.
+* (**Deprecated**) [Original pirate codelab written using dart:html][client-codelab], where you learn to build a web app. This lab should take about an hour to complete.
+* (**Needs updates**) [Beware the Nest o' Pirates][server-codelab] which shows you how to write a RESTful Dart server.
```

Comment:
Let's add a new paragraph linking to all the codelabs:

For links to the latest Dart codelabs, see [dartlang.org/codelabs](https://www.dartlang.org/codelabs).